packager/mpd/base/xml/xml_node.cc

```diff
@@ -47,7 +47,8 @@ std::string RangeToString(const Range& range) {
 // Check if segments are continuous and all segments except the last one are of
 // the same duration.
 bool IsTimelineConstantDuration(const std::list<SegmentInfo>& segment_infos,
-                                uint32_t start_number) {
+                                uint32_t start_number,
+                                const double target_duration) {
```

Comment:
There is no need to pass |target_duration| to this function with the new code.

Reply:
Have made the above change. Sorry about that.